README.md

```diff
@@ -117,7 +117,7 @@ export default Ember.Controller.extend({
 ## FullCalendar Scheduler
 
 ### Opting In
-By default, FullCalendar Scheduler is NOT imported. To include it, add the following to your application's `ember-cli-build.js`:
+By default, FullCalendar Scheduler is NOT imported. To include it, add the following to your application's `config/environment.js`:
```

Comment:
I should have separate the issues, sorry.
If I put `includeScheduler` option in `ember-cli-build.js`, it does not get imported. When I put it in `config/environment`, it does.
Actually, in your `index.js`, which config does this line load ? I admit Ember internals are still a bit of mystery to me :-)
```javascript
var config = target.project.config(target.env) || {};
```
I'm using ember-cli 2.11, perhaps the behavior is not the same as 2.12 ?
I updated the docs to fix the instructions as you mentioned. The fork is working with ember-cli 2.11.